addons/adminMenu/XEH_postInit.sqf

```diff
@@ -1,5 +1,25 @@
 #include "script_component.hpp"
 
+[QGVAR(resetGear), {
+    params ["_unit"];
+    diag_log text format ["[POTATO] Reseting Gear on %1 [%2]", (name _unit), _unit];
+    if ((isNull _unit) || {!alive _unit} || {!local _unit}) exitWith {};
+
+    removeAllWeapons _unit;
+    removeGoggles _unit;
+    removeHeadgear _unit;
+    removeVest _unit;
+    removeUniform _unit;
+    removeAllAssignedItems _unit;
+    removeBackpack _unit;
+
```

Comment:
1. we shouldn't be removing goggles here.
2. shouldn't the gear script do this anyways? (I know it won't double execute, but seems a bit wasteful)